fhir-search/src/test/java/com/ibm/fhir/search/date/DateTimeHandlerTest.java

```diff
@@ -579,7 +580,7 @@ public void testYearMonthDaySeparatorHourMinutesSecondsMicrosecondsParserZoneBad
     @Test(expectedExceptions = FHIRSearchException.class)
     public void testYearMonthDaySeparatorHourMinutesSecondsMicrosecondsParserZoneBadZoneUseWithDay()
             throws FHIRSearchException {
-        // This case is 
+        // This case is
```

Comment:
probably better just deleting this one :-)
```suggestion
```

Reply:
Removed.